test-suite/src/data/schema.ts

```diff
@@ -222,94 +222,95 @@ const schema: Schema = {
         },
         {
           testName: "Safeguarding Referral",
-          testDescription: "This test will attempt to send notification of a safeguarding referral to Social Services.",
+          testDescription:
+            "This test will attempt to send notification of a safeguarding referral to Social Services.",
           template: require("./safeguarding-request.xml"),
           testSpecifications: {
             [RequestHeaderProps.Header]: [
               {
                 label: "url",
                 id: "url",
                 defaultValue: "http://localhost:8081/report",
-                validators: [isUrl(), isReportUrl()]
+                validators: [isUrl(), isReportUrl()],
```

Comment:
the comma is not required at the end here.

Reply:
I was of the the same opinion, it's enforced by the prettier formatter.

Reply:
I've made changes to the prettier config as discussed.